### PR TITLE
Add new Formula, Grakn.ai

### PR DESCRIPTION
**Pull request by merqurio**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by merqurio** [comment]
working on the rubocop problem... not sure why is failing

File: Formula/grakn.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  depends_on :java => "1.8+"
+
+  def install
+    # Install jars in libexec to avoid conflicts
+    libexec.install Dir["*"]
+
+    # Symlink binaries
+    bin.install Dir["#{libexec}/bin/*"]
+
+    # Set where the executable is
```
Comment (MikeMcQuaid) [inline Formula/grakn.rb]: You can remove all these comments, they are self-explanatory, thanks!

File: Formula/grakn.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    Web UI:
+    http://localhost:4567
+
+    Data should not be affected on upgrade.
+    EOS
+  end
+
+  test do
+    assert_match /Engine has stopped\nCassandra has stopped/i, shell_output("#{bin}/grakn.sh status", 0)
```
Comment (MikeMcQuaid) [inline Formula/grakn.rb]: The `0` second argument is unnecessary as it's the default. Shorten the regex or split this onto two lines so it fits in 80 characters. Thanks!

File: Formula/grakn.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    To open the REPL:
+    $ graql.sh
+
+    Web UI:
+    http://localhost:4567
+
+    Data should not be affected on upgrade.
+    EOS
+  end
```
Comment (MikeMcQuaid) [inline Formula/grakn.rb]: You can remove all these caveats as they are mostly not Homebrew specific, thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by merqurio** [comment]
Thanks for the suggestions @MikeMcQuaid, should be ok now.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**